packages/lab/stories/input-next/input-next.doc.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { InputNext } from "@salt-ds/lab";
+import { Meta, Canvas, ArgsTable } from "@storybook/addon-docs";
+import {
+  CSSClassTable,
+  CharacteristicUsage,
+} from "css-variable-docgen-components";
+import { Link } from "@salt-ds/core";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+<Meta
+  title="Documentation/Lab/Input Next"
+  component={InputNext}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Input Next
+
+Input provides an editable field in which users can enter arbitrary text and numeric values.
+
+# Examples
+
+## Default
+
+Input comprises value text and a focus ring. The component is initially empty and then, once it has focus, it accepts text on a single line only.
+
+<Canvas>
+  <Story id="core-input-next--default" />
+</Canvas>
+
+## Controlled
+
+You can control Input with the `value` and `onChange` props. Learn more about <Link href="https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components">controlled vs uncontrolled components in the official React docs</Link>.
+
+<Canvas>
+  <Story id="core-input-next--controlled" />
+</Canvas>
+
+## Variants
+
+Input has two variants, primary (default) and secondary.
+
+<Canvas>
+  <Story id="core-input-next--variants" />
+</Canvas>
+
+## Disabled
+
+You can set Input to a disabled state, with no resultant action when the user interacts with it.
+
+<Canvas>
+  <Story id="core-input-next--disabled" />
+</Canvas>
+
+## Readonly
+
+You can set Input to a read-only state. The value text can’t be edited in this state, but it can be highlighted and copied.
+
+<Canvas>
+  <Story id="core-input-next--readonly" />
+</Canvas>
+
+For accessibility purposes, it is recommended that you add a marker for empty read only fields via the `emptyReadOnlyMarker` prop.
+
+<Canvas>
+  <Story id="core-input—next--empty-readonly-marker" />
+</Canvas>
+
+## Text Alignment
+
+Text alignment is left by default. This can be changed via the `textAlign` prop to “center” or “right”.
+
+<Canvas>
+  <Story id="core-input—next--text-alignment" />
+</Canvas>
+
+## Validation
+
+Input can show validation states (warning, success, error) with the prop `validationStatus`.
+
+<Canvas>
+  <Story id="core-input-next--validation" />
+</Canvas>
+
+- Use the error state to alert the user of a critical issue that’s related to the input. This issue, which may jeopardize completion of the task, usually requires action from the user to resolve the error.
+- Display the warning state when you need to alert the user of a potential issue—that won’t prevent the user from continuing with the task, but may cause errors if it’s not addressed.
+
+# API
+
+```
+import { InputNext } from "@salt-ds/lab";
```

Comment:
This should be at the beginning, no?

Reply:
hmm, it's in this position in all other docs + template

Reply:
Eesh, ok, then leave it as is 👍

Reply:
worth raising it with the content team :)

Reply:
I think it's fine to keep it here so it aligns to the template